bin/cgp-pass.sh

```diff
@@ -22,27 +22,39 @@ done
 
 parse_network "$NETWORK"
 
+if [ "$NETWORK" = "alfajores" ]; then
+    RPC_URL="http://127.0.0.1:8545"
+    echo "🔗 Using local RPC URL $RPC_URL"
+fi
```

Comment:
maybe add a comment here like in the setup.sh explaining why do we need to do this?